src/Page/App.elm

```diff
@@ -27,6 +38,20 @@ body model =
 
 view : Model -> Html Msg
 view model =
+    let
+        isAdmin =
+            case model.client of
```

Comment:
I'd probably have written the check like this:
```elm
isAdmin =
  model.client
    |> Maybe.andThen (\c -> c.username)
    |> Maybe.map(\username -> username == "admin") -- or Maybe.map ((==) "admin")
    |> Maybe.withDefault False
```
Doesn't matter much in this case, but I'm just curious if you know what the best practice in Elm is.